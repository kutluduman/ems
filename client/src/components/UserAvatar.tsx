import Avatar from "@mui/material/Avatar";

import { stringAvatar } from "../utils";

interface UserAvatarProps {
  name?: string;
  surname?: string;
}

const UserAvatar = ({ name, surname }: UserAvatarProps) => {
  return <Avatar {...stringAvatar(`${name} ${surname}`)} />;
}
export default UserAvatar;