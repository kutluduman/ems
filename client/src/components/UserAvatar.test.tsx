import { create } from "react-test-renderer";

import UserAvatar from "./UserAvatar";

describe("UserAvatar Component", () => {
  test("renders the UserAvatar Component", () => {
    const tree = create(<UserAvatar name="John" surname="Doe" />);

    expect(tree.toJSON()).toMatchSnapshot();
  });
});