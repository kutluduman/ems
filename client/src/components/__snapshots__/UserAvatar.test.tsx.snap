// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserAvatar Component renders the UserAvatar Component 1`] = `
<div
  className="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-16za3gy-MuiAvatar-root"
>
  JD
</div>
`;
